packages/prettier-config/lib/index.js

```diff
@@ -13,7 +13,7 @@ const prettierPackage = require( require.resolve( 'prettier/package.json' ) );
 
 const isWPPrettier = prettierPackage.name === 'wp-prettier';
 const customOptions = isWPPrettier
-	? { parenSpacing: true, jsxBracketSameLine: false }
+	? { parenSpacing: true, bracketSameLine: false }
```

Comment:
`bracketSameLine` is the same now so we can move it to the `config` object.

Reply:
Cleaned up :+1: